public/app/features/explore/DrilldownAlertBox.tsx

```diff
@@ -9,9 +9,14 @@ type Props = {
 };
 
 export function DrilldownAlertBox(props: Props) {
-  const isDsCompatibleWithDrilldown = ['prometheus', 'loki', 'tempo', 'grafana-pyroscope-datasource'].includes(
-    props.datasourceType
-  );
+  const isDsCompatibleWithDrilldown = [
+    'prometheus',
+    'grafana-amazonprometheus-datasource',
```

Comment:
Would these new data source be automatically available to select in Metrics Drilldown?

Reply:
Yes, they would be available to select in Metrics Drilldown.